main.go

```diff
@@ -46,12 +46,13 @@ func main() {
 	flag.StringVar(location, "l", "40.748,-73.985", "`LOCATION` to be queried (shorthand)")
 	numdays := flag.Int("days", 3, "`NUMBER` of days of weather forecast to be displayed")
 	flag.IntVar(numdays, "d", 3, "`NUMBER` of days of weather forecast to be displayed (shorthand)")
-	unitSystem := flag.String("units", "metric", "`UNITSYSTEM` to use for output.\n    \tChoices are: metric, imperial, si, metric-ms")
-	flag.StringVar(unitSystem, "u", "metric", "`UNITSYSTEM` to use for output. (shorthand)\n    \tChoices are: metric, imperial, si, metric-ms")
```

Comment:
Whats the idea here for migrating older wegorc's to the new settings? I still think we should keep the compatibility with setting a single unit for all three instead of just having the per-type unit. At least in the settings file it still makes sense, no?

Reply:
I wanted to keep it initially for backward compatibility. But then thought "what for", it seems less clear to have all four options at the same time. 

Also we have to decide which takes precedence over which one:

- Global unit overrides individual units : Great backward compatibility and not disrupt existing users configuration. But it means we would have to allow an empty string as global unit to allow per unit configuration.

- Individual units override global unit : Standard way. But ingo would create the new units with default values in the configuration file. And the existing global unit wouldn't be used anyway. Or we rather allow empty values for individual units as default.

What do you think, how should we proceed then?